docs/v1/open_questions.md

```diff
@@ -0,0 +1,112 @@
+# Open Questions
+
+This document contains open questions regarding the
+[API design document](./design.md).
+
+## 1. Is the `ServiceInstanceBinding` resource necesssary?
+
+The `ServiceInstanceBinding` object is a byproduct of the successful execution
+of a `ServiceInstanceClaim`. Instead of writing a new `ServiceInstanceBinding`
+object, can the service catalog controller just add the binding information
+directly into the `ServiceInstanceClaim` on success?
+
+## Should we write Kubernetes references for binding information?
+
+Currently, the information in a `ServiceInstanceBinding` that points to created
+resources looks like this:
+
+```yaml
+secrets:
+  - my-zip-code-creds-password
+configMaps:
+  - my-zip-code-creds-conn-info
+```
+
+Instead, should we write pointers to these resources using Kubernetes reference
+notation, as below?
+
+```yaml
+resources:
+  - /api/v1/namespaces/${NAMESPACE}/secrets/my-zip-code-creds-password
```

Comment:
The representation of references should be driven by their intended usage.

We represent almost no resource references as paths currently, because we've optimized for writing them, with some contextual information (e.g., namespace) inferred and omitted, rather than using them verbatim in clients.